### ISSUE TITLE
An option to autoplay next episode with x amount of seconds remaining in current episode?

### ISSUE THREAD

**Issue by cboswald95** [im85288/service.upnext#211]
I'm not sure if this is still actively developed, but I and others would love an option to enable autoplay of the next episode with a set amount of seconds remaining in the one currently being watched.  

This is now how Netflix's next episode feature functions, and it's quite useful. It would make the Kodi experience feel much more professional.

**Comment by im85288** [comment]
Yes still being developed but as everyone has a day job not always super fast. The feature you requested has already been discussed in depth. We all agree it would be a nice feature but it will only really work when integrated with the netflix addon...this is because the netflix addon is the only integration that is aware of exactly when the credits should appear. 

@dagwieers and others know more of the details than I do. 

I'll leave this open for now, but it will most likely be closed as being a duplicate. Have a look at some of the open PRs and I am sure you'll find the discussion I was referring to,
